### ISSUE TITLE
Can someone help me with my load path?

### ISSUE THREAD

**Issue by howdoicomputer** [jorgenschaefer/emacs-buttercup#62]
I'm not sure if I'm doing this correctly.

My directory structure is:

```
emacs-habitica\
    tests\
         test-feature.el
    feature.el
```

Except when I run buttercup:

```
drfra@spacecat ~Users\drfra\workspace\emacs-habitica master $ cask exec buttercup -L .
Cannot open load file: no such file or directory, feature
```

I was following the wiki... am I doing something incorrectly?

I was able to get around this by using the `load-relative` library.

```
(require 'load-relative)

(require-relative "../feature")
```

That made it work but it doesn't feel "right".

**Comment by jorgenschaefer** [comment]
Hello! You're not doing anything wrong, that should just work.

Does `emacs -L . -Q -batch --eval "(require 'feature)"` work?

**Comment by howdoicomputer** [comment]
Hm, I don't get anything but I am on Windows.

```
drfra@spacecat ~Users\drfra\workspace\emacs-habitica master # emacs -L . -Q -batch --eval "(require 'feature)"
drfra@spacecat ~Users\drfra\workspace\emacs-habitica master #
```

Just a returned newline. Lemme' grab my mac...

**Comment by jorgenschaefer** [comment]
That means it worked, no need to test further.

**Comment by howdoicomputer** [comment]
Well, `cask exec buttercup -L .` just failed on Windows *but* it passes on OS X. Hm.

**Comment by jorgenschaefer** [comment]
> Well, `cask exec buttercup -L .` just failed on Windows but it passes on OS X. Hm.

Aha! … I just set up a test directory structure here, and it works fine, but I do not have a Windows system available to test this, sadly :-(

If you find out what's wrong there and if buttercup can do anything to support Windows better, please do let me know.

**Comment by howdoicomputer** [comment]
Yeah, no problem. I did add the batch script. I am just learning Emacs Lisp so it'll take me a while to fix this.

I wonder if this block of code has trouble with Windows based paths?

https://github.com/jorgenschaefer/emacs-buttercup/blob/master/buttercup.el#L684-L687

**Comment by jorgenschaefer** [comment]
Hm. You can add `(message "Loading %s" file)` in front of the `(load ...)`  line and see if that runs at all, and what file it tries to load.

**Comment by jcs090218** [comment]
I am also getting this issue, this is what work for me.

In function `buttercup-run-discover`:

```diff
- (dolist (dir (or dirs '(".")))
+ (dolist (dir (or dirs `(,default-directory)))
```

I think the `"."` isn't safe enough; use `default-directory` should be safer IMO.

Or alternatively, you can do:

```diff
- (load file nil t))))
+ (load (expand-file-name file) nil t))))
```

They are pretty much the same.
